code/lib/test/template/stories/loader-enhancements.stories.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable @typescript-eslint/naming-convention,storybook/prefer-pascal-case */
+import { expect, within, userEvent as globalUserEvent } from '@storybook/test';
+
+const meta = {
+  component: globalThis.Components.Button,
+  parameters: { chromatic: { disable: true } },
+  args: { label: 'Button' },
+};
+
+export default meta;
+
+export const canvas_is_equal_to_within_canvas_element = {
+  async play({ canvas, canvasElement }) {
+    const oldCanvas = within(canvasElement);
+    await expect(canvas satisfies typeof oldCanvas).toEqual(within(canvasElement));
```

Comment:
I think this is cleaner:
```suggestion
    await expect(canvas satisfies typeof oldCanvas).toEqual(oldCanvas);
```
Not sure why it's named `oldCanvas` though.